docs/developers/meta/sc-config.md

```diff
@@ -82,7 +88,38 @@ The settings are as follows:
     - _default_: `[]`
 - `kill-legacy-callback`
     - The framework has no way of knowing whether or not a certain smart contract variant actually needs the async callback code or not, so in rare situations it is necessary to forcefully remove it.
-    - _values_: `true` | `false
+    - _values_: `true` | `false`
+    - _default_: `false`
+- `codegen-units`
+    -  Controls the number of "code generation units" a crate will be split into. Splitting a crate into multiple code generation units can have a significant impact on the compile time and code optimization of the crate.
```

Comment:
Add:
- From our experience it is of no particular impact to smart contract compilation.